frontend/app/views/fragments/form/terms.scala.html

```diff
@@ -2,17 +2,16 @@
 @import com.gu.i18n.CountryGroup
 @import configuration.CopyConfig.usLegalTerms
 
-@(countryGroup: Option[CountryGroup] = None, monthlyContribution: Boolean = false)
+@(countryGroup: CountryGroup, monthlyContribution: Boolean = false)
 
 <p class="ts-and-cs">@if(monthlyContribution){By proceeding,} else {By joining Guardian Members,} you are agreeing to our
-    <a href="@if(monthlyContribution){@Links.monthlyContributionTerms}else{@Links.membershipTerms(countryGroup)}" class="text-link" target="_blank">Terms and Conditions</a> and
+    <a href="@if(monthlyContribution){@Links.monthlyContributionTerms}else{@Links.membershipTerms(Some(countryGroup))}" class="text-link" target="_blank">Terms and Conditions</a> and
     <a href="@Links.guardianPrivacyPolicy" class="text-link" target="_blank">Privacy Policy</a>.
 </p>
 
-@countryGroup.map{ cg =>
-    @if(Seq("us","um","vi").contains(cg.id.toLowerCase)) {
-        <div class="us-legal">@usLegalTerms</div>
-    } else {
-        <div class="us-legal is-hidden">@usLegalTerms</div>
-    }
+
+@if(Seq("us","um","vi").contains(countryGroup.id.toLowerCase)) {
```

Comment:
Note that is line is a little dodgy: https://github.com/guardian/membership-frontend/pull/1375/files#r119852653